ner_v1/chatbot/entity_detection.py

```diff
@@ -349,7 +349,7 @@ def get_phone_number(message, entity_name, structured_value, fallback_value, bot
                                   bot_message=bot_message)
 
 
-def get_email(message, entity_name, structured_value, fallback_value, bot_message):
+def get_email(message, entity_name, structured_value, fallback_value, bot_message, is_asr):
```

Comment:
For my understanding, when do we send structured value and what does it hold?

Reply:
Structured Value is a string obtained from any structured elements like UI elements like form, payload, etc. 
If structured value is present detection is run on structured value instead of message usually. That's handled by the BaseDetector

Reply:
Understood. Thanks for explaining.